pdoc/render_helpers.py

```diff
@@ -336,23 +336,27 @@ def linkify_repl(m: re.Match):
         else:
             # It's not, but we now know the parent module. Does the target exist?
             doc = context["all_modules"][module]
-            if qualname:
-                assert isinstance(doc, pdoc.doc.Module)
-                doc = doc.get(qualname)
-            target_exists_and_public = (
-                doc is not None and context["is_public"](doc).strip()
-            )
-            if target_exists_and_public:
-                assert doc is not None  # mypy
+            # Check again if the object was re-exposed
+            for module, qualname in possible_sources(
```

Comment:
I _think_ this should use `qualname_candidates` here.

Reply:
`qualname` is a string in this case, so I don't think it should be `candidates`